changelog-template.hbs

```diff
@@ -0,0 +1,17 @@
+
+{{#each releases}}
+
+  ## [{{title}}] {{date}}
+  {{#if summary}}
+    {{summary}}
+  {{/if}}
+  {{#each merges}}
+    - {{#if commit.breaking}}**Breaking change:** {{/if}}{{message}}{{#if href}} [`#{{id}}`]({{href}}){{/if}}
+  {{/each}}
+  {{#each fixes}}
+    - {{#if commit.breaking}}**Breaking change:** {{/if}}{{commit.subject}}{{#if href}} [`#{{id}}`]({{href}}){{/if}}
```

Comment:
You don't need to backtick-quote the pull request ID or commit hash. GitHub will handle it just fine without.